lib/vagrant-vmware-desktop/driver/base.rb

```diff
@@ -398,7 +398,11 @@ def create_vmnet_device(config)
         # This deletes the VM.
         def delete
           @logger.info("Deleting VM: #{@vm_dir}")
-          @vm_dir.rmtree
+          begin
+            @vm_dir.rmtree
+          rescue Errno::ENOTEMPTY
+            FileUtils.rm_rf(@vm_dir.to_s)
```

Comment:
Why not always `FileUtils.rm_rf(@vm_dir.to_s)` instead of trying `@vm_dir.rmtree` and falling back to using FileUtils?

Reply:
Mainly just to retain existing behavior, since it rarely happens :shrug: